src/tshirtsize.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+namespace TShirtSize{
+    char getTShirtSizeName(int shoulderSizeIncms);

```

Comment:
Why not make this like sensorstub.h and leave out the `get` from the function name?